frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -286,6 +297,12 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
           backButtonText={t('ceph-storage-plugin~Back')}
           onSave={finalStep}
           onClose={() => history.goBack()}
+          onNext={({ id }) => {
+            const idIndexPlusOne = Object.values(CreateStepsBC).indexOf(CreateStepsBC[id]) + 1;
+            const newStepHigherBound =
+              stepHigherBound < idIndexPlusOne ? idIndexPlusOne : stepHigherBound;
+            setStepHigherBound(newStepHigherBound);
+          }}
```

Comment:
```suggestion
    setStepsReached(StepIndexMap[id]);
```
If id is the upcoming step then we might have to add some guard for corner cases.

Reply:
I just tested out the changes and it seems to work well. I think we are good here since `onNext` will only be called for `n-1` steps and `onSave` will be called for the last one.